### PR TITLE
Feature/history: Folder Location, Regeneration, and appending to current git history

### PR DESCRIPTION
**Pull request by Sylk**
3 new features added. Updated the docs to reflect the changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Generate Git Commits
 
-A command-line tool that generates GitHub or GitLab activity graph to make it look like you have been coding regularly.
+A command-line tool that generates GitHub or GitLab activity graph to make it look like you have been coding incredibly irregularly and to make you look like a machine.
```
Comment (artiebits) [inline README.md:3]: i would prefer to keep "regularly" :)

File: src/index.js [hunk 2]
```diff
@@ -16,42 +16,70 @@ const chalk = require("chalk");
...
+    // Create ALL of the commits
     for (const date of commitDateList) {
       // Change spinner so user can get the progress right now.
       const dateFormatted = new Intl.DateTimeFormat("en", {
         day: "numeric",
         month: "long",
         year: "numeric"
       }).format(date);
-      spinner.text = `Generating your Github activity... (${dateFormatted})\n`;
+      spinner.text = `Generating a distinguished github history full of lies... 😈 (${dateFormatted})\n`;
```
Comment (artiebits) [inline src/index.js:82]: let's leave this line as is coz "full of lies" sounds little big negative

**Review by artiebits — COMMENTED** [review]
thank you very much for your contribution! I have a few small comments, the rest looks fine to me

File: README.md (hunk 1 above)
Comment (Sylk) [reply README.md:3]: How about a compromise and saying "you have been coding on an incredibly frequent basis."

The definer of regularly would mean there's a pattern to the commits, but we're randomizing them.

File: src/index.js (hunk 2 above)
Comment (Sylk) [reply src/index.js:82]: I mean, this is a tool that's used to create lies. Why not lean into the evil a bit, and have some fun with it?

Comment (Sylk) [reply src/index.js]: rEVERTED THIS CHANGE.

**Comment by Sylk** [comment]
Alright, I removed my fun verbiage back to what you wanted. Cheers mate.

**Review by Brandonagil — APPROVED** [review]

**Comment by Sylk** [comment]
Why was this never merged? This has full-scope automation so that you can run this as a daily cron.

File: README.md [hunk 3]
```diff
@@ -57,6 +57,12 @@ However, if you want to generate activity for specific dates, use these options:
 npx fake-git-history --startDate "2020/09/01" --endDate "2020/09/30"
 ```
 
-## PS
+### `--regenerate, -r`
+This option is defaulted to `false` however if you would like to regenerate your github history project, you can use this to force it to regenerate it with a full new history while simultaneously blasting the old one away.
 
-This tool was created as a joke, so please don't take it seriously. While cheating is never encouraged, if someone is judging your professional skills based on your GitHub activity graph, they deserve to see a rich activity graph 🤓
+### `--historyFolder, -h `
```
Comment (artiebits) [inline README.md:63]: i would propose using `-f` since `-h` is commonly used for "help"

File: src/index.js [hunk 4]
```diff
@@ -16,42 +16,70 @@ const chalk = require("chalk");
...
+    // Create ALL of the commits
     for (const date of commitDateList) {
       // Change spinner so user can get the progress right now.
       const dateFormatted = new Intl.DateTimeFormat("en", {
         day: "numeric",
         month: "long",
         year: "numeric"
       }).format(date);
-      spinner.text = `Generating your Github activity... (${dateFormatted})\n`;
+      spinner.text = `Generating a distinguished github. (${dateFormatted})\n`;
```
Comment (artiebits) [inline src/index.js:82]: i would like to keep the original message, thanks

**Review by artiebits — COMMENTED** [review]
@Sylk  I am sorry about this. For some time I havent had any interest in supporting my open-source projects anymore. Let's revisit this PR. I have a couple of requests tho, I hope you can address them. Thanks for your contribution!

**Pull request closed without merging**